doc/fitting/livefit_logger.md

```diff
@@ -0,0 +1,52 @@
+# Fitting Files Callback
+## Fitting Files
+
+The callback (`LiveFitLogger`) exists to write all fitting metrics from `LiveFit` to file.
+
+This callback provides you with useful metrics such as `R-squared` and `chi-square`, then providing you with a table of the raw collected data included modelled `y` data and `y` uncertainty. 
+
+### Example
+An example of using this could be: 
```

Comment:
Let's talk about this but maybe there's ways to even out the ratio of explanation-to-code a bit.

Otherwise I'm worried that with large code blocks like this in the docs we're just duplicating our implementation again...